app-emulation/spice/spice-0.15.0.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="The SPICE client and server library"
+DESCRIPTION="The SPICE package provides the SPICE server library and client.
+
+These components are used to provide access to a remote machine's \
+display and devices"
+HOMEPAGE="https://www.spice-space.org/"
+COPYRIGHT="2009 Red Hat, Inc"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.spice-space.org/download/releases/spice-server/spice-$portVersion.tar.bz2"
+CHECKSUM_SHA256="b320cf8f4bd2852750acb703c15b72856027e5a8554f8217dfbb3cc09deba0f5"
+SOURCE_DIR="spice-$portVersion"
+PATCHES="spice-$portVersion.patchset"
+
+ARCHITECTURES="all"
```

Comment:
I suppose we want to disable x86_gcc2?

Reply:
This recipe(spice) provides libspice-server for qemu, they should have the same platforms supported, I will check